open-source-trumps-innersource.md

```diff
@@ -0,0 +1,35 @@
+## Title
+Open Source trumps InnerSource
+
+## Statement of Problem
+* People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
```

Comment:
It's not clear to me that this behavior is a problem.

Reply:
I think the core problem is that people's mindset is that open source is always superior to InnerSource, which often is not the case. The result is that this bias leads people to select and use inferior components, perhaps having to put effort to adapting them that would otherwise go towards improving and reusing the InnerSource component.

Reply:
I see.  I think that point would be useful to roll into the statement - the inner source component meets their needs better than an open source component.  They should choose the component that best meets their needs irrespective of it being open source vs. inner source?

Reply:
This feedback has been worked into the "Problem" section already. Resolving.

@gruetter if you would like to see your statement about COTS integrated into the pattern, would you mind sending a suggestion for this?